build/run-backup.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+request_backup() {
+	curl -s http://${SRC_NODE}:6033/backup/${BACKUP_NAME} -o /backup/${BACKUP_NAME}.stream
```

Comment:
should we have the same checks for PSO https://github.com/percona/percona-docker/blob/main/percona-xtradb-cluster-8.0-backup/run_backup.sh#L153-L160 ?

Reply:
I'd like to have this checks but it requires installing a separate client for each cloud provider. It'd be very nice if xbcloud have a `stat` command.

Reply:
please create the task for it

Reply:
https://jira.percona.com/browse/K8SPS-90